client/my-sites/stats/site.jsx

```diff
@@ -232,7 +232,10 @@ class StatsSite extends Component {
 									<InlineSupportLink
 										supportContext="stats"
 										showIcon={ false }
-										showSupportModal={ !! config( 'is_running_in_jetpack_site' ) }
+										showSupportModal={
+											config.isEnabled( 'is_running_in_jetpack_site' ) ||
+											config( 'is_running_in_jetpack_site' )
```

Comment:
the `|| config( 'is_running_in_jetpack_site' )` should be removed once https://github.com/Automattic/jetpack/pull/27471 is deployed.